boards/remote-revb/include/periph_conf.h

```diff
@@ -26,29 +26,12 @@
 #include "cc2538_gpio.h"
 #include "periph_cpu.h"
 #include "periph_common.h"
+#include "periph/gpio.h"
```

Comment:
Needed?

Reply:
I don't think so, might be an artefact from rebasing